services/common/flags.go

```diff
@@ -686,6 +691,7 @@ func DefaultConfig(homeDir string) *config.Config {
 		SubmitNoSynced:       false,
 		DevNextGDB:           true,
 		GBTTimeOut:           defaultGBTTimeout,
+		SnapTimeout:          300,
```

Comment:
Better to add extra `defaultSnapTimeout`  after line 43 : `defaultGBTTimeout = 800`